specification/search/resource-manager/Microsoft.Search/preview/2019-10-01-preview/examples/CreateOrUpdateSharedPrivateLinkResource.json

```diff
@@ -0,0 +1,41 @@
+{
+  "parameters": {
+    "searchServiceName": "mysearchservice",
+    "resourceGroupName": "rg1",
+    "api-version": "2019-10-01-preview",
+    "subscriptionId": "subid",
+    "sharedPrivateLinkResourceName": "testResource",
+    "sharedPrivateLinkResource": {
+      "name": "testResource",
+      "properties": {
+        "requestMessage": "please approve",
+        "groupId": "blob",
+        "privateLinkResourceId": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Storage/storageAccounts/storageAccountName"
+      }
+    }
+  },
+  "responses": {
+    "200": {
```

Comment:
why do we have both 200 and 201 here?

Reply:
Because @arv100kri requested it earlier in this PR to show both the create and update scenario.